internal/controllers/suite_test.go

```diff
@@ -62,13 +68,22 @@ func setup(reader templates.Reader, helm Helm, objects []runtime.Object) (*testi
 	if err != nil {
 		return nil, err
 	}
+	// Setup event recorder
+	kubeClient, err := kubernetes.NewForConfig(cfg)
```

Comment:
no need to use `kube-config` flag and this function. There is `ctrl.GetConfigOrDie()`, it does similar thing

Reply:
well, it's already there, and `cfg` is smaller than `ctrl.GetConfigOrDie()`

Reply:
and seems we need it because of kubebuilder